css/game.css

```diff
@@ -0,0 +1,90 @@
+* {
+    box-sizing: border-box;
+}
+
+.game-wrapper {
```

Comment:
nothing confining the height of this box to the agreed dimensions

Reply:
fixed in 8000153